src/unstract/adapters/x2text/llm_whisperer/src/llm_whisperer.py

```diff
@@ -72,18 +75,15 @@ def __make_request(
         files = None
```

Comment:
I think, Without files params/ or None we don't need to call LLM whisperer .Insetad we can raise some error from here self. Don't want in this PR. we can make it later also filne.